### PR TITLE
Icon sweep

### PR DESCRIPTION
**Pull request by drw158**
This resolves some issues and adds new icons. This is easier to do all at once, rather than running grunt for each icon in separate PRs

Icons added and/or changed:

Mic #147 
Mention #172 
Offline #179 
Resize #180 
Add User #21 
Fullscreen #40 
Fullscreen exit #40 
filter #185 

Please visit each issue or PR for design discussion

screen shot 2017-01-18 at 10 16 01 am

### CODE REVIEW

**Comment by drw158** [comment]
Sorry y'all for the large PR. I wanted to get many lingering icon issues closed. Less painful to do a separate PR for each icon :)

I opened the index.html file and all seemed well.

File: php/gridicons.php [hunk 1]
```diff
@@ -314,7 +323,7 @@ function get_gridicon( $gridicon ) {
 			$svg = '<svg class="gridicon gridicons-menus" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M9 19h10v-2H9v2zm0-6h6v-2H9v2zm0-8v2h12V5H9zm-4-.5c-.828 0-1.5.672-1.5 1.5S4.172 7.5 5 7.5 6.5 6.828 6.5 6 5.828 4.5 5 4.5zm0 6c-.828 0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5 1.5-.672 1.5-1.5-.672-1.5-1.5-1.5zm0 6c-.828 0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5 1.5-.672 1.5-1.5-.672-1.5-1.5-1.5z"/></g></svg>';
 			break;
 		case 'gridicons-microphone':
-			$svg = '<svg class="gridicon gridicons-microphone" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M12 13c1.104 0 2-.896 2-2V6c0-1.104-.896-2-2-2-1.105 0-2 .896-2 2v5c0 1.104.895 2 2 2zm4-2c0 2.21-1.79 4-4 4s-4-1.79-4-4H6c0 2.972 2.164 5.433 5 5.91V20h2v-3.09c2.836-.478 5-2.94 5-5.91h-2z"/></g></svg>';
+			$svg = '<svg class="gridicon gridicons-microphone" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><title>gridicons-microphone</title><path d="M19 9v1a7 7 0 0 1-6 6.92V20h3v2H8v-2h3v-3.08A7 7 0 0 1 5 10V9h2v1a5 5 0 0 0 10 0V9zm-7 4a3 3 0 0 0 3-3V5a3 3 0 0 0-6 0v5a3 3 0 0 0 3 3z"/></g></svg>';
```
Comment (drw158) [inline php/gridicons.php]: @jasmussen Does the script now insert the title for new icons in this php file?

File: svg-min/gridicons-resize.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/><g><title>gridicons-resize</title><path d="M13 4v2h3.59L6 16.59V13H4v7h7v-2H7.41L18 7.41V11h2V4h-7"/></g></svg>
```
Comment (drw158) [inline svg-min/gridicons-resize.svg]: @jasmussen I noticed that in the newest version of AI, it includes the title in the svg when you export. It looks like the minification script does not explicitly strip out the title, so it gets included in the generated icons.

**Comment by folletto** [comment]
Design wise 👍  to all.

**Comment by drw158** [comment]
@jasmussen Can you review this once you get a few minutes. Thanks!

**Comment by jasmussen** [comment]
These all look good to me. 

Incidentally I noticed that the Offline icon now got a title, which none of the other icons did. But since we need that anyway (see https://github.com/Automattic/gridicons/issues/186), I think that's fine.

**Comment by drw158** [comment]
Thanks Joen. It's strange, the new AI export options include `title` and the grunt task does not strip it out. If it's fine to leave in there, then I'll merge!

**Pull request merged**